tern/analyze/common.py

```diff
@@ -222,6 +223,59 @@ def convert_to_pkg_dicts(pkg_dict):
     return pkg_list
 
 
+def get_deb_package_license(deb_copyright):
+    '''
+    Given the debian copyright file location,
+    1. parse the copyright file
+    2. filter out all the available licenses
+    3. returns a list of unique licenses found inside
+    the copyright text
+    '''
+    mounted_path = os.path.join(
+        rootfs.mount_dir, constants.temp_folder, constants.mergedir)
+    deb_copyright = '{0}{1}'.format(mounted_path, deb_copyright)
+    deb_pkg_data = DebianCopyright.from_file(deb_copyright).to_dict()
```

Comment:
From the above inference, I don't think we can use this. And it looks like debut doesn't have a direct way of instantiating this class given just a debian copyright text. However, there also seems to be `debcon.get_paragraphs_data` [here](https://github.com/nexB/debut/blob/972761071d70a2665a434c9df83fdaf371c338ee/src/debut/debcon.py#L502) which we can possibly use directly. It might be worthwhile to look into this first.

Reply:
@nishakm the link which you pointed out to `get_paragraph_data_from_file` was my earlier proposal in the notebook but you suggested going in this way.

Reply:
Yes, `get_paragraph_data_from_file` reads a file and then calls `get_paragraphs_data` on the read text. This is the functionality I would rather use, instead of referring to the file path. This helps with our separation of how individual OSs choose to record this metadata. I haven't tested out `get_paragraphs_data` but you could try it out and see if it works.

Reply:
Done! Please review the update file.